app/controllers/sidebar_sections_controller.rb

```diff
@@ -39,6 +37,7 @@ def update
     @guardian.ensure_can_edit!(sidebar_section)
 
     sidebar_section.update!(section_params.merge(sidebar_urls_attributes: links_params))
+    sidebar_section.sidebar_section_links.update_all(user_id: sidebar_section.user_id)
```

Comment:
Since we're running two related DB commands, I think it'll be better to wrap this in the same transaction as the above.